browser-features/pages-clips/src/globals.css

```
@@ -0,0 +1,4 @@
+@import "tailwindcss";
+@plugin "daisyui" {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)